src/main/features/first-contact/guards/claimExpiredGuard.ts

```diff
@@ -0,0 +1,17 @@
+import * as express from 'express'
+import { ErrorPaths } from 'first-contact/paths'
+import { User } from 'idam/user'
+
+const logger = require('@hmcts/nodejs-logging').getLogger('first-contact/guards/claimReferenceMatchesGuard')
+
+export class ClaimExpiredGuard {
```

Comment:
this needs renaming, something like `ClaimantRequestedCCJGuard` or `ClaimantAlreadyRequestedCCJGuard`

Reply:
Update.
(I went with ClaimantRequestedCCJGuard)